src/Resolution.ts

```diff
@@ -168,12 +168,16 @@ export default class Resolution {
    * @async
    * @param domain - domain name to be resolved
    * @param currencyTicker - currency ticker like BTC, ETH, ZIL
+   * @deprecated since Resolution v1.7.0
    * @returns A promise that resolves in an address or null
    */
   async address(
     domain: string,
     currencyTicker: string,
   ): Promise<string | null> {
+    console.warn(
+      'Resolution#address is deprecated since v1.7.0, use Resolution#addr instead',
+    );
```

Comment:
Spamming users' console on each deprecated call isn't a very good idea, imo. I think we should either remove it or make sure it's printed only once

Reply:
I think it is a good idea in this case. That wouldn't be a good idea if there would be multiple warnings per user action. But it is unlikely. 

In my experience, warn once is a good practice when you can warn on application boot. Warn once per use action is not that great, but better than warn on first user action because that error message can be lost too easily.

Reply:
I believe writing into console is really invasive and isn't that necessary. Just to imagine two cases where I'd hate having a library that spams into my stdout:
- I have a logging system that relies on certain format of messages (for instance, we use winston logger ourselves, and we try hard to avoid anything printed via `console.log`)
- I make really a lot of `address` calls, and my own messages get lost in deprecation warnings

I still strongly vote for changing this behavior @bogdan @JohnnyJumper @DeRain